diff --git a/static/js/lib/api.js b/static/js/lib/api.js
index 8c487032..f0531a5e 100644
--- a/static/js/lib/api.js
+++ b/static/js/lib/api.js
@@ -12,6 +12,12 @@ import type { AvailableProgram, AvailablePrograms } from '../flow/enrollmentType
 import type { EmailSendResponse } from '../flow/emailTypes';
 import type { PearsonSSOParameters } from '../flow/pearsonTypes';
 import { fetchWithCSRF, fetchJSONWithCSRF } from 'redux-hammock/django_csrf_fetch';
+import {
+  DASHBOARD_RESPONSE,
+  PROGRAMS,
+  COURSE_PRICES_RESPONSE,
+  USER_PROGRAM_RESPONSE,
+} from '../test_constants.js';
 
 const loginOnEdXError = (response: Response) => {
   if (response.errorStatusCode === 400 || response.errorStatusCode === 401) {
@@ -37,13 +43,14 @@ export function patchUserProfile(username: string, profile: Profile): Promise<Pr
 }
 
 export async function getDashboard(username: string): Promise<Dashboard> {
-  try {
-    let response = await fetchJSONWithCSRF(`/api/v0/dashboard/${username}/`);
-    return response;
-  } catch (response) {
-    loginOnEdXError(response);
-    return response;
-  }
+  // try {
+  //   let response = await fetchJSONWithCSRF(`/api/v0/dashboard/${username}/`);
+  //   return response;
+  // } catch (response) {
+  //   loginOnEdXError(response);
+  //   return response;
+  // }
+  return DASHBOARD_RESPONSE
 }
 
 export function checkout(courseId: string): Promise<CheckoutResponse> {
@@ -90,13 +97,14 @@ export function sendLearnerMail(subject: string, body: string, studentId: number
 }
 
 export async function getPrograms(): Promise<AvailablePrograms> {
-  try {
-    let response = await fetchJSONWithCSRF('/api/v0/programs/');
-    return response;
-  } catch (response) {
-    loginOnEdXError(response);
-    return response;
-  }
+  // try {
+  //   let response = await fetchJSONWithCSRF('/api/v0/programs/');
+  //   return response;
+  // } catch (response) {
+  //   loginOnEdXError(response);
+  //   return response;
+  // }
+  return PROGRAMS;
 }
 
 export function getProgramLearners(programId: number): Promise<ProgramLearners> {
@@ -141,10 +149,11 @@ export function addFinancialAid(income: number, currency: string, programId: num
 }
 
 export function getCoursePrices(username: string): Promise<CoursePrices> {
-  return fetchJSONWithCSRF(`/api/v0/course_prices/${username}/`).then(coursePrices => {
-    // turn `price` from string into decimal
-    return R.map(R.evolve({price: Decimal}), coursePrices);
-  });
+  // return fetchJSONWithCSRF(`/api/v0/course_prices/${username}/`).then(coursePrices => {
+  //   // turn `price` from string into decimal
+  //   return R.map(R.evolve({price: Decimal}), coursePrices);
+  // });
+  return Promise.resolve(COURSE_PRICES_RESPONSE);
 }
 
 export function skipFinancialAid(programId: number): Promise<*> {
